integration/spark/src/main/scala/org/apache/spark/sql/execution/command/mutation/CarbonProjectForUpdateCommand.scala

```diff
@@ -190,6 +192,10 @@ private[sql] case class CarbonProjectForUpdateCommand(
             updateTableModel,
             executionErrors)
 
+          // clear invalid segments from cache
+          IndexStoreManager.getInstance()
```

Comment:
instead of handling this in command class, better move this to `DeleteExecution.deletedeltaexecution` and will be used from both update and delete flow

Reply:
Done